ui/src/app/apps/apps-add/uri/apps-bulk-import-uri.component.html

```diff
@@ -0,0 +1,44 @@
+<app-page-head class="step2">
+  <app-page-head-back [defaultUrl]="'/apps/add'" [isNotRegex]="'^(\/apps\/add\/)'"></app-page-head-back>
+  <app-page-head-title>Import Application(s) <strong>from URI</strong></app-page-head-title>
+</app-page-head>
+
+<div dataflowLayoutType type="large">
+  <form [formGroup]="form" role="form" (ngSubmit)="submit()" novalidate>
+
+    <div class="dataflow-alert dataflow-alert-info">
+      Provide a <strong>URI</strong> that points to the location of the <strong>properties file</strong>. <br>
+      This properties file is formatted as keys are the `type.name` and the values are the URIs of the apps.
```

Comment:
Change

```
This properties file is formatted as keys are the `type.name` and the values are the URIs of the apps.
```

to (something like this, to wordy?)

```
This properties file is formatted so that the keys represent the <i>type</i>
and the <i>name<i/> of the application, e.g. <strong>type.name</strong>. The property values are the URIs of the app. The file should look like the following example:

sink.file=maven://org.springframework.cloud.stream.app:file-sink-rabbit:1.3.1.RELEASE
source.ftp=maven://org.springframework.cloud.stream.app:ftp-source-rabbit:1.3.1.RELEASE
processor.transform=maven://org.springframework.cloud.stream.app:transform-processor-rabbit:1.3.1.RELEASE
task.timestamp-batch=maven://org.springframework.cloud.task.app:timestamp-batch-task:1.0.0.RELEASE
```